api/src/pcapi/core/offerers/repository.py

```diff
@@ -575,6 +575,21 @@ def get_offerer_bank_accounts(offerer_id: int) -> models.Offerer | None:
     - Linked venues to its bank accounts (possibly none and only current ones)
     - Managed venues by the offerer (possibly none)
     """
+    has_non_free_offers_subquery = db.session.query(
```

Comment:
On prend en compte les offres gratuites, donc le nommage prête à confusion.
```suggestion
    venue_has_offers_subquery = db.session.query(
```

Reply:
bien vu merci